### PR TITLE
(depreciated) Improvment for the first phase of zktrie

### PR DESCRIPTION
**Pull request by noel2004**
This work has finished the first part of a better PR for zktrie: inducing zktrie as an alternative storage layer and providing a more effective, unitify logging scheme for the L2 trace.

There is some breaking change for the trace output comparing to current zktrie work branch:

1. All the "proof" parts are purged from "extraData" field
2. The tracing for "after" status of SSTORE opcode is removed, and so "CaptureStateAfter" is of no use now

I think these changes has no affection on the trace handling of roller because no module would refer the proof part except for the mpt-circuit

### CODE REVIEW

**Comment by lispc** [comment]
I suggest you open another PR (squash all commits into 1) against zkrollup branch. And prefix "depreciated" to title of this one. Dont close this one. So we can have a look at diff for   both base branches

**Comment by noel2004** [comment]
Ok, one moment

> I suggest you open another PR (squash all commits into 1) against zkrollup branch. And prefix "depreciated" to title of this one. Dont close this one. So we can have a look at diff for both base branches

File: core/blockchain.go [hunk 1]
```diff
@@ -1188,8 +1188,15 @@ func (bc *BlockChain) writeKnownBlock(block *types.Block) error {
 	return nil
 }
 
+// EvmTxTraces groups trace data from each executation of tx and left
+// some field to be finished from blockResult
+type EvmTxTraces struct {
```
Comment (mask-pp) [inline core/blockchain.go]: Moving `EvmTxTraces` to `core/types/l2trace.go` might be more concentrated.

Comment (noel2004) [reply core/blockchain.go]: get it, may us just move to https://github.com/scroll-tech/go-ethereum/pull/101 for more reviewing?

**Pull request closed without merging**